tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,133 @@
+import SectionSummaryPage from "../generated_pages/list_collector_section_summary_items/section-summary.page.js";
+
+describe("List Collector Section Summary Items", () => {
+  describe("Given I launch the test list collector section summary items survey", () => {
+    beforeEach(() => {
+      browser.openQuestionnaire("test_list_collector_section_summary_items.json");
+    });
+    it("When I answer yes to the driving question, Then I should be able to add my own item and the relevant data about it and get to the relevant page.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect(browser.getUrl()).to.contain(AnyCompaniesOrBranchesPage.url());
+    });
+    it("When I add my own item and relevant data, Then after submitting I should be asked if I have more items to add.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect($(AnyCompaniesOrBranchesPage.yes()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.no()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.heading()).getText()).to.contain(
+        "Do you need to add any other UK companies or branches that undertake general insurance business?"
+      );
+    });
+    it("When I add my own item and relevant data, Then after I answer no on additional items page I should get to the section summary page.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      $(AnyCompaniesOrBranchesPage.no()).click();
+      $(AnyCompaniesOrBranchesPage.submit()).click();
+      expect(browser.getUrl()).to.contain(SectionSummaryPage.url());
+    });
+    it("When I add my own item and relevant data, Then after I answer yes on additional items page I should be able to choose an item from the same pre-defined list and add relevant data about it.", () => {
```

Comment:
What does pre-defined mean here? Should be like this>
```suggestion
    it("When I add my own item and relevant data, Then after I answer yes on additional items page I should be able to choose an item from the items list and add relevant data about it.", () => {
```

Reply:
I took it from the acceptance criteria list and forget to change it to be more specific after, thanks!